lib/DDG/Spice/RiffsyEmojiSearch.pm

```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::RiffsyEmojiSearch;
```

Comment:
Let's namespace these: `DDG::Spice::Riffsy::EmojiSearch` and `DDG::Spice::Riffsy::GifSearch`

@bryanhart due to the way we're using Pull Requests with our Community Platform, I actually need you to add the 2nd Perl Package (DDG::Spice::Riffsy::GifSearch) in a new Pull Request though.

Sorry about the extra work! I'm going to make a task to add this to the docs!